### PR TITLE
V2 Bigtable API

### PR DESCRIPTION
**Pull request by nevillelyh**
This addresses #56

### CODE REVIEW

**Comment by nevillelyh** [comment]
@rgruener FYI if you wanna try it out.

**Comment by rgruener** [comment]
You want me to try it before its merged?

**Comment by nevillelyh** [comment]
Should that'll be cool.

File: scio-examples/src/main/scala/com/spotify/scio/examples/extra/BigtableV2Util.scala [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  : (ByteString, Iterable[Mutation]) =
+    (kc.encode(key), Iterable(setCell(familyName, columnQualifier, value)))
+
+  def setCell[F, C, V](familyName: F, columnQualifier: C, value: V)
+                      (implicit fc: Coder[F], cc: Coder[C], vc: Coder[V]): Mutation =
+    Mutation.newBuilder().setSetCell(
+      SetCell.newBuilder()
+        .setFamilyNameBytes(fc.encode(familyName))
+        .setColumnQualifier(cc.encode(columnQualifier))
+        .setValue(vc.encode(value))
```
Comment (ravwojdyla) [inline scio-examples/src/main/scala/com/spotify/scio/examples/extra/BigtableV2Util.scala]: `.setTimestampMicros(-1)` for BT server timestamp

**Pull request merged**